pyntrinio/pyntrinio.py

```diff
@@ -189,14 +189,54 @@ def gather_stock_time_series(api_key, ticker, start_date, end_date, output_forma
   Example
   -----------
   >>> gather_stock_time_series(api_key, 'AAPL')
-  {'ticker':['AAPL'], 'adj_close':[300], 'date':[datetime.date(2020, 1, 2)]}
   """
+  try:
+    # change dates to datetime objects
+    if start_date is not None:
+      start_date = datetime.strptime(start_date, '%Y-%m-%d').date()
+    if end_date is not None:
+      end_date = datetime.strptime(end_date, '%Y-%m-%d').date()
+  except:
+    print("Invalid Date format - please input the date as a string with format %Y-%m-%d")
+    return
   
```

Comment:
Here we might also want to check if the input ticker is as required.   
```suggestion
  if type(ticker) != str:
      print("Invalid Input: `ticker` has to be a single `str`, e.g. 'APPL' ")
      return
```

Reply:
Added! Good suggestion!